zoo-css-challenge/index.html

```diff
@@ -35,37 +36,39 @@ <h2>Bears</h2>
 					src="https://placebear.com/200/300"
 					alt="Grizzly Bear"
 					width="200"
-					height="300"
+					height="400"
 				>
 				<img src="https://placebear.com/400/300"
 					alt="Grizzly Bear"
 					width="400"
-					height="300"
+					height="400"
+					
 				>
 				<img src="https://placebear.com/200/400"
 					alt="Grizzly Bear"
 					width="200"
 					height="400"
+					
 				>
 			</div>
 		</section>
 
 		<section class="tigers">
-			<h2>Tiger</h2>
-			<h3>Top 5 Tiger Facts</h3>
-			<ol>
+			<h2>‘Tiger Facts’</h2>
+			
+			<ul>
 				<li>Tigers are the largest cat species in the world reaching up to 3.3 meters in length and weighing up to 670 pounds!
 				<li>Tigers are easily recognizable with their dark vertical stripes and reddish/orange fur.
 				<li>The Bengal tiger is the most common tiger.
 				<li>Tigers live between 20-26 years in the wild.
 				<li>Unlike most other cats, tigers are great swimmers and actually like the water.
-			</ol>
+			</ul>
 		</section>
 
 		<section class="giraffe">
 			<h2>Giraffe</h2>
 
-			<p>Giraffes are the world's tallest mammals, thanks to their towering legs and long necks. A giraffe's legs alone are taller than many humans—about 6 feet . These long legs allow giraffes to run as fast as 35 miles an hour over short distances and cruise comfortably at 10 miles an hour over longer distances.</p>
+			<p><strong>Giraffes are the world's tallest mammals, thanks to their towering legs and long necks. A giraffe's legs alone are taller than many humans—about 6 feet . These long legs allow giraffes to run as fast as 35 miles an hour over short distances and cruise comfortably at 10 miles an hour over longer distances.</strong></p>
```

Comment:
```suggestion
			<p>Giraffes are the world's tallest mammals, thanks to their towering legs and long necks. A giraffe's legs alone are taller than many humans—about 6 feet . These long legs allow giraffes to run as fast as 35 miles an hour over short distances and cruise comfortably at 10 miles an hour over longer distances.</p>
```
The task is to make the font size of this paragraph larger, which we can do with the CSS property `font-size`. `<strong>` makes the font bold, which is not related to font size.